src/hooks/useRestaurantReviews.ts

```diff
@@ -0,0 +1,37 @@
+import { ListReviews } from '../client/flavorite/models'
+import { useQuery } from 'react-query'
+import { GetReviewByRestaurantRequest, RestaurantsApi } from '../client/flavorite/apis'
+
+export default function useRestaurantReviews(restaurantId: GetReviewByRestaurantRequest) {
+  const Restaurants = new RestaurantsApi()
+  const fetchRestaurantReviews = Restaurants.getReviewByRestaurant(restaurantId)
+  const Reviews = useQuery(['reviewsByRestaurant', restaurantId], () => fetchRestaurantReviews)
```

Comment:
This may make many requests when the hook is called. 
```suggestion
  const fetchRestaurantReviews = () => Restaurants.getReviewByRestaurant(restaurantId)
  const Reviews = useQuery(['reviewsByRestaurant', restaurantId], fetchRestaurantReviews)
```
This way useQuery would call fetch as we want.